dev/io.openliberty.pages.3.1.internal.factories/bnd.bnd

```diff
@@ -25,28 +22,28 @@ Bundle-Description: WAS Pages 3.1, version ${bVersion}
 WLP-ServerName: IBM WebSphere Application Server
 WLP-ServerVersion: ${liberty.beta.version}
 
--dsannotations: com.ibm.ws.pages31.webcontainerext.JspVersionFactoryImpl, \
-com.ibm.ws.pages31.webcontainerext.ws.PrepareJspHelper23Factory, \
-com.ibm.ws.pages31.translator.visitor.validator.ElValidatorExtFactoryImpl, \
-com.ibm.ws.pages31.translator.visitor.generator.GeneratorUtilsExtFactoryImpl
+-dsannotations: com.ibm.ws.jsp23.webcontainerext.JspVersionFactoryImpl, \
+com.ibm.ws.jsp23.webcontainerext.ws.PrepareJspHelper23Factory, \
+com.ibm.ws.jsp23.translator.visitor.validator.ElValidatorExtFactoryImpl, \
+com.ibm.ws.jsp23.translator.visitor.generator.GeneratorUtilsExtFactoryImpl
 
 # Note these factories were copied from the com.ibm.ws.jsp.2.3 bundle
 
 Export-Package: \
 	com.ibm.ws.jsp.speclevel, \
-	com.ibm.ws.pages31.webcontainerext;version="3.1", \
-	com.ibm.ws.pages31.webcontainerext.ws;version="3.1", \
-	com.ibm.ws.pages31.translator.visitor.validator;version="3.1", \
-	com.ibm.ws.pages31.translator.visitor.generator;version="3.1", \
+	com.ibm.ws.jsp23.webcontainerext;version="3.1", \
+	com.ibm.ws.jsp23.webcontainerext.ws;version="3.1", \
+	com.ibm.ws.jsp23.translator.visitor.validator;version="3.1", \
+	com.ibm.ws.jsp23.translator.visitor.generator;version="3.1", \
 	org.apache.jasper;version="3.1", \
 	org.apache.jasper.el;thread-context=true;version="3.1", \
 	org.apache.jasper.runtime;thread-context=true;version="3.1"
 
 Import-Package: \
-   !com.ibm.ws.pages31.translator.visitor.generator, \
-   !com.ibm.ws.pages31.translator.visitor.validator, \
-   !com.ibm.ws.pages31.webcontainerext, \
-   !com.ibm.ws.pages31.webcontainerext.ws, \
+   !com.ibm.ws.jsp23.translator.visitor.generator, \
+   !com.ibm.ws.jsp23.translator.visitor.validator, \
+   !com.ibm.ws.jsp23.webcontainerext, \
+   !com.ibm.ws.jsp23.webcontainerext.ws, \
```

Comment:
Since we didn't change these classes between 2.3, 3.0, and 3.1, I changed the package back to jsp23 which is what exists in the `com.ibm.ws.jsp.2.3` bundle.  Not need to create overlays just for a package change.

Reply:
I should reformat this section.